docker/data_source_docker_plugin.go

```diff
@@ -0,0 +1,75 @@
+package docker
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceDockerPlugin() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceDockerPluginRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"alias": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Docker Plugin alias.",
```

Comment:
You might want to remove all '.' from the end of the descriptions

Reply:
makes sense @suzuki-shunsuke  to keep in line with the other descriptions.

Reply:
it makes sense @suzuki-shunsuke to keep in line with the other descriptions.

Reply:
Addressed. e3384e9401656632da53dfcd42b1f06c3562847b